src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -14,9 +14,9 @@ import { useProfileModalControls } from "hooks/useProfileModalControls";
 
 import { RoomModal } from "components/templates/PartyMap/components";
 
-import { NavSearchBarInput } from "./NavSearchBarInput";
-
 import "./NavSearchBar.scss";
+import { InputField } from "components/atoms/InputField";
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
```

Comment:
These imports should be sorted differently (3rd party import at the top, `components/atoms/*` below `components/templates/*`